CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ As opposed to being able to test on Google Cloud you can easily contribute to th
 
 #### Setup
 1. Install [Apache Airflow](https://airflow.apache.org/start.html) and [Apache Oozie](https://oozie.apache.org/docs/4.3.1/DG_QuickStart.html) locally
```

Comment:
It sounds like setup.py will install Airflow for us. Do we need users to install airflow?